src/main/kotlin/com/vk/kphpstorm/inspections/quickfixes/ReplaceToKphpFunctionsQuickFix.kt

```diff
@@ -0,0 +1,34 @@
+package com.vk.kphpstorm.inspections.quickfixes
+
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.openapi.project.Project
+import com.jetbrains.php.lang.lexer.PhpTokenTypes
+import com.jetbrains.php.lang.psi.PhpPsiElementFactory
+import com.jetbrains.php.lang.psi.elements.FunctionReference
+
+class ReplaceToKphpFunctionsQuickFix(private val maybeFunctionName: String) : LocalQuickFix {
```

Comment:
I don't think this is the best field name here.

Reply:
Fixed